ocean_provider/utils/test/test_util.py

```diff
@@ -54,15 +53,8 @@ class Dummy:
     content_type = mimetypes.guess_type(filename)[0]
     url_object = {"url": f"https://source-lllllll.cccc/{filename}", "type": "url"}
     _, instance = FilesTypeFactory.validate_and_create(url_object)
-    with patch(
-        "ocean_provider.file_types.definitions.is_safe_url",
-        side_effect=[True],
-    ):
-        with patch(
-            "requests.get",
-            side_effect=[mocked_response],
-        ):
-            response = instance.build_download_response(request)
+    with patch("requests.get", side_effect=[mocked_response]):
```

Comment:
I'm do not agree with mocking the request.get. It is too generic and we risk "testing that our test works". Also, this looks like one of our tests, not an arweave-specific one. It failed? Why would adding the arweave integration warrant changes in the other tests?

Reply:
Mocking `requests.get` existed before my changes. I did not add this.

The test failed because I removed the `is_safe_url` check from `build_download_response` and `is_safe_url` nor longer exists at the `ocean_provider.file_types.definitions.is_safe_url` scope.

> - Remove `is_safe_url` check from `build_download_response`
>   * When calling download, `is_safe_url()` is called earlier by `check_details()` so calling again in `build_download_response` is redundant.
>   * When calling computeResult, `validate_url=False` so `is_safe_url()` check is skipped anyway.

@calina-c As I mentioned in my previous comment, I'd rather keep these changes but if it's a deal-breaker, please let me know and I'll revert them.

Reply:
yes, please keep it for now; we can create an issue to follow-up and discuss on removing is_safe_url, separately.